src/cutapplier.cc

```diff
@@ -21,10 +21,10 @@ using std::cout;
 using std::endl;
 
 int main(int argc, char *argv[]) {
-	if(argc != 5 ){
+	if(argc != 5 && argc != 6){
 		cout << "cutapplier:   	Applies a cut to an input ntuple"<< endl;
```

Comment:
You don't produce output that tells the user of the intention of the friend tree: Is the friend tree merged into the output ntuple after applying cuts? A more informative description of what the optional argument does would be useful here.